packages/shared/components/popups/VoteForProposalPopup.svelte

```diff
@@ -62,7 +63,9 @@
         {/if}
     </div>
     <popup-buttons class="flex flex-row flex-nowrap w-full space-x-4">
-        <Button classes="w-full" outline onClick={closePopup}>{localize('actions.cancel')}</Button>
+        <Button classes="w-full" disabled={isTransferring} outline onClick={closePopup}
```

Comment:
Same as above^

Reply:
Because cancel to the user means that it will cancel an action, if you have already kicked off the action then the cancel button is not cancelling anything.

So its confusing to the user if it doesn't do anything. Why would we have a cancel button that only closes a popup when there is a close button.